src/store/reducers/ActionCreators.ts

```diff
@@ -0,0 +1,100 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const BASE_URL = 'https://mate.academy/students-api';
+
+export const fetchUsers = createAsyncThunk(
+  'user/fetchAll',
+  async (_, thunkAPI) => {
+    try {
+      const url = `${BASE_URL}/users`;
+      const response = await fetch(url);
+
+      return (await response.json()) as User[];
```

Comment:
It's not good practice using `as`.
Read more about fetch typification [here](https://kentcdodds.com/blog/using-fetch-with-type-script)